pom.xml

```diff
@@ -36,6 +36,11 @@
         <hibernate-validator.version>6.1.6.Final</hibernate-validator.version>
         <jackson.version>2.10.5</jackson.version>
 
+        <!-- Sonar Properties -->
+        <sonar.projectKey>kiwiproject_dropwizard-elk-appender</sonar.projectKey>
```

Comment:
Oh, is this how it is working? Do we want these properties here in the POM vs. in the build workflow? Are we able to run Sonar locally and have it send results to SonarCloud?

Reply:
We could run it locally if we have the token, yes.  It is just a maven command.  We aren't actually using the Sonar specific action because we are using maven.  The sonar action is specifically for languages running the sonar-scanner cli.